aptos/examples/mint_nft/README.md

```diff
@@ -0,0 +1,19 @@
+# Example Full-Stack App: 100$ USD Mint
+
+The goal of this contract is managing an NFT mint where the mint is paid in native currency but the cost of one NFT is always 100$.
+This example is intended to be run on Aptos testnet because it depends on Pyth and Wormhole existing onchain.
+
+### How to deploy the smart contract :
+
+- Use `aptos init` with rest_url : `https://testnet.aptoslabs.com/` and faucet `https://faucet.testnet.aptoslabs.com`  to generate a new keypair.
+- Use a faucet to airdrop testnet APT to your newly created account by calling `aptos account fund-with-faucet --account default`. If this doesn't work, I have had success importing my private key from `.aptos/config.yaml` into Petra and clicking the airdrop button. Otherwise send APT from another account.
+- Get your account address from `.aptos/config.yaml` and replace `mint_nft="_"` by `mint_nft="<ADDRESS>"` in `Move.toml`
+- `aptos move compile`
+- `aptos move publish` 
+
+### How to run the webapp :
+
+- In `app/src/App.tsx` replace `const MINT_NFT_MODULE = "_"` by `const MINT_NFT_MODULE = "<ADDRESS>"` the address of your module from above.
+- `npm install`
+- `npm run start`
+- Go to `http://localhost:3000/` in your browser and use Petra wallet to transact with the app.
```

Comment:
Suggest adding a section like "Repository Organization" that points the reader to the files that are particularly interesting / relevant (i.e., the contract and the file with the frontend logic)

Reply:
i don't love the title "repository organization" but something like that

Reply:
Sure